gradle/libs.versions.toml

```diff
@@ -51,6 +51,10 @@ aqute-bnd = "6.4.1"
 asciidoctor-mathjax = "0.0.9"
 # @keep Asciidoctor tabs version used in ref-guide
 asciidoctor-tabs = "1.0.0-beta.6"
+azure-storage = "12.25.0"
+azure-identity = "1.12.0"
+azure-core = "1.52.0"
+azure-core-http-netty = "1.15.4"
```

Comment:
I see this PR adds a bunch of netty httpclient dependencies. We already have Netty as a dependency with version 4.2.6 but azure brings in v4.1.110 in addition. 

Either force the use of 4.2.6 for azure-blob if possible, or select another http-client, switch to the `azure-core-http-jdk-httpclient` client or make a wrapper around the jetty client (probably a tall order).

Reply:
Did you have a chance to look at the http client?